src/components/Text/Text.st.css

```diff
@@ -11,6 +11,10 @@
     RunningTextFont: "{theme: 'Body-M', size: '16px', lineHeight: '1.5em'}";
     ListTextFont: "{theme: 'Body-M', size: '16px', lineHeight: '2em'}";
     LargeTitleFont: "{theme: 'Heading-M', size: '32px', lineHeight: '1.25em'}";
+    MobileSmallTitleFont: "{theme: 'Page-title', size: '20px', lineHeight: '1.4em'}";
```

Comment:
Are these variables necessary? I think it might add unnecessary complexity. Users could still override the font using the `:mobile` state in their `.st.css` files.

Reply:
this variables represent the design system specs. users can override everything in there `.st.css` with or without the `mixin`

Reply:
The way I see it, practically speaking, the style variables give the developers a way to change the binding of Wix's styleParams to their own settings panel. Mobile sites don't have a settings panel at the moment, and so most of the developers won't need to use these variable.
Theoretically speaking, the text is the same text element, and so the variable should stay the same. 
If we'll see that it's missing for developers in the future I guess we could add it, but for now it seems redundant.
It might seem a bit petty, but I think it will give a clearer api for developers.
@shlomitc I'd love to hear what you think